CondTools/RPC/data/RPCTwinMuxLinkMapInput.txt

```diff
@@ -0,0 +1,300 @@
+TM YB+1_S1 0 LB_RB+1_S1_BP1A 1580	CH0: W+1/RB1out/1 Backward 	CH1: W+1/RB2out/1 Backward 	CH2: W+1/RB1in/1 Backward 
```

Comment:
Most likely never, worst case scenario it could happen during a long shutdown period.  I wasn't aware of the option of a repository for this kind of data, if you have a link to some documentation it would be most welcome for this or future applications.